cli/src/main.rs

```diff
@@ -52,6 +52,225 @@ pub fn main() {
         }
     };
 
+    let regtest = matches.is_present("regtest");
+    if regtest {
+        use std::ffi::OsString;
+        use std::fs::File;
+        use std::io;
+        use std::path::PathBuf;
+        use std::process::{Command, Stdio};
+        use std::{thread, time};
```

Comment:
Nice!  I like proximal to use, especially for this.  Are we sure that none of these are redundant?

Reply:
Right now they're not all in use, but there are plans to use the `std::io` for example, IIRC, to log `zcashd` `stdout`.